samples/suit/smp_transfer/templates/extflash/rad_envelope.yaml.jinja2

```diff
@@ -0,0 +1,106 @@
+{%- set mpi_radio_vendor_name = application['config']['CONFIG_SUIT_MPI_RADio_LOCAL_1_VENDOR_NAME']|default('nordicsemi.com') %}
```

Comment:
Aren't these two manifests (radio and root) just copies of suit-generator templates? Why keep them as separate files?

Reply:
Those are not exact copies. The manifest has fetch sequence.